config.yml.dist

```diff
@@ -0,0 +1,12 @@
+oauth2:
+  clients:
+    client-name:
+      id: id
+      secret: secret
+      redirect_uri: http://www.example.com/
+  server:
+    authorize_uri: http://www.example.com/auth
+    token_uri: http://www.example.com/token
+    client_id: client_id
+    client_secret: client_secret
+    scope: profile email
```

Comment:
Maybe shouldn't be in config?

Reply:
Meant shouldn't be configurable, ie in code. (Profiles will be interacting with ORCID, not Journal etc.)

Reply:
The `scope` itself could be a constant somewhere as it is likely the code would stop working if that config is changed?